src/Razor/src/Microsoft.NET.Sdk.Razor/build/netstandard2.0/Sdk.Razor.CurrentVersion.targets

```diff
@@ -374,11 +374,16 @@ Copyright (c) .NET Foundation. All rights reserved.
     -->
     <Warning Text="Unable to resolve a Razor configuration for the value '$(RazorDefaultConfiguration)'. Available configurations '@(RazorConfiguration->'%(Identity)', ', ')'.'"
       Code="RAZORSDK1000"
-      Condition="'@(ResolvedRazorConfiguration->Count())' == '0'" />
+      Condition="'$(_Targeting30OrNewerRazorLangVersion)' == 'true' AND '@(ResolvedRazorConfiguration->Count())' == '0'" />
```

Comment:
`RazorDefaultConfiguration` isn't defined for a 2.1 project when restore hasn't completed but there's still some design time builds that kick off (❓). You get an unnecessary warning in that case. Limiting the audience to 3.0 or newer projects